torch/_dynamo/eval_frame.py

```diff
@@ -74,6 +76,11 @@ class Unset(Enum):
 
 compile_lock = threading.RLock()
 most_recent_backend: Optional[CompilerFn] = None
+DONT_WRAP_FILES = {
+    # For tracing into fx modules
+    inspect.getsourcefile(GraphModule),
+    join(dirname(dirname(__file__)), "onnx/_internal/fx/dynamo_exporter.py"),
```

Comment:
Hi @jansel , could you kindly explain what this does and why it is needed after this PR? :) My impression is this is somewhat related to the wrapping I did in that file, but unclear on the details. Much appreciated.

Reply:
@BowenBao currently `onnx/_internal/fx/dynamo_exporter.py` is relying on incorrect behavior of `torch._dynano.export()`.   I would very much like to delete this line since it is incorrect.

ONNX is doing something like:
```
def wrapper(...):
   ...

torch._dynamo.export(wrapper, ...)
```

If you wote that code normally, dynamo would try to capture the contects of `wrapper()` like any normal function.

However, that doesn't happen because ONNX is a subdirectory of `torch/*`, which is listed in `skipfiles.py`.  The following check:
https://github.com/pytorch/pytorch/blob/b9146d8b0bbda045a19b564a2081919eaf78582d/torch/_dynamo/skipfiles.py#L167
causes the outer `wrapper()` to be completely ignored by dynamo, and dynamo just captures the inner frame called by `wrapper()`.  I have no idea why this works, or how ONNX depends on this incorrect behavior, but it does.

You could try instead adding this file to:
https://github.com/pytorch/pytorch/blob/b9146d8b0bbda045a19b564a2081919eaf78582d/torch/_dynamo/skipfiles.py#L109
Which will cause `export()` to process the wrapper rather than ignore it.